ntbs-service/Migrations/20191003090016_AddTbServiceColumnsForAdGroups.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class AddTbServiceColumnsForAdGroups : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "PHECAdGroup",
```

Comment:
Is there a reasonable max length on the size that this could be?

Reply:
Good spot!
Looks like there is: https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2003/cc756101(v=ws.10)?redirectedfrom=MSDN#fqdn-length-limitations
AD domain names are limited to 64 characters.